graphql/schemabuilder/pagination.go

```diff
@@ -149,6 +149,14 @@ type PaginationInfo struct {
 	Pages          []string
 }
 
+// PostProcessOptions can be returned in a PaginateFieldFunc as an argument that succeeds
+// PaginationInfo. When a PaginationInfo is set, it must be succeeded by a PostProcessOptions.
+// The struct is used to instruct Thunder to perform additional operations on the output
+// in the case of manually paginated resolvers.
```

Comment:
nit: I feel that the last sentence here is the most important, since it explains what this struct means, so maybe that should be the first sentence in the comment.